src/gwc/src/main/java/org/geoserver/gwc/config/GWCConfig.java

```diff
@@ -31,6 +35,14 @@
     private boolean TMSEnabled;
     
     private boolean securityEnabled;
+    
+    private boolean innerCachingEnabled;
```

Comment:
Some comments, here or better yet as as javadocs on the accessor method methods to explain what "Inner Caching" and "Persistence" are would be helpful.